lib/function/probability/factorial.js

```diff
@@ -55,31 +55,36 @@ module.exports = function (math) {
 
       n = n.toNumber();   // Should definitely be below Number.MAX_VALUE  
       if (n < smallBigFacs.length) {
-        return BigNumber.convert(smallBigFacs[n]).toSD(math.precision);
+        return BigNumber.convert(smallBigFacs[n]).toSD(config.precision);
       }
 
+      // Be wary of round-off errors
+      var precision = config.precision + 4;
```

Comment:
I just wanted a log that would give an extra `2` at the start. I can be more conservative and go with just `Math.log(n)`.